cmd/ndm_daemonset/probe/eventhandler.go

```diff
@@ -178,6 +187,10 @@ func generateUUID(bd blockdevice.BlockDevice) (string, bool) {
 		uuidField = bd.DeviceAttributes.WWN
 		ok = true
 	case len(bd.PartitionInfo.PartitionTableUUID) > 0:
+		// PartitionTable UUID is used if the disk contains only the partition table and not a partition. This is a very rare
```

Comment:
If the NDM identifies the disk using partition table uuid:
(a) Will the device path still be /dev/sda?
(b) If given to Local PV. Local PV will format the device with filesystem. On reboot / restart of the ndm will uuid be generated correctly?
(c) If given to cStor, cStor will create partitions. On reboot/restart of NDM, will uuid be detected correctly?

Reply:
yes, a) the device path will still be `/dev/sda` 
b) If formatted it won't work,
c) If partitions are created, exisiting partition table UUID will be used.

In that case, shall we update the design to remove partition table UUID based identification. and always use partition entry

Reply:
update code to remove `PartitionTableUUID` from algorithm used for UUID generation.